nservicebus/msmq/scaling-out_overview_core_6.partial.md

```diff
@@ -16,4 +16,6 @@ Following table compares both approaches:
 | Round-robin load balancing                  | Fair load balancing                         |
 
 
-WARNING: When using sender-side distribution scaling out technique in a mixed version environment make sure to deploy a distributor in front of the scaled out version 6 endpoint if that endpoint needs to communicate with endpoints using versions lower than 6 (refer to [the distributor sample](/samples/scaleout/distributor/) for details). Otherwise event messages will get duplicated and commands won't be properly distributed.
\ No newline at end of file
+WARNING: When using sender-side distribution scaling out technique in a mixed version environment, make sure to deploy a distributor in front of the scaled out version 6 endpoint if that endpoint needs to communicate with endpoints using versions lower than 6 (refer to [the distributor sample](/samples/scaleout/distributor/) for details). Otherwise event messages will get duplicated and commands won't be properly distributed.
```

Comment:
How about changing the warning to make it more clear? Suggestion below:

WARNING: To avoid duplicate messages, i.e. events or commands from being processed by multiple worker instances when using the Sender-Side distribution feature, it is important that a distributor is deployed in front of the endpoint. This restriction only applies in a mixed version environment, i.e. if the endpoint is running Versions 6 and above and is communicating with other endpoints running Versions 5 or below.  For more details, refer to the [distributor sample.](/samples/scaleout/distributor/)